### ISSUE TITLE
Failed to load the remote-ftp package

### ISSUE THREAD

**Issue by funcvar** [icetee/remote-ftp#1222]
**Atom**: 1.32.2 x64
**Electron**: 2.0.9
**OS**: Ubuntu 18.04.1
**Thrown From**: [remote-ftp](https://github.com/icetee/remote-ftp) package 2.2.1

### Stack Trace

Failed to load the remote-ftp package

```
At variable @ui-size is undefined in /home/oleg/.atom/packages/remote-ftp/styles/remote-ftp.less:172:13

LessError: variable @ui-size is undefined
at /home/oleg/.atom/packages/remote-ftp/styles/remote-ftp.less:172:13
```

**Comment by re-sign** [comment]
The problem was solved by changing @ui-size to @font-size. In my situation it was in non-default theme atom-material-ui.

**Comment by icetee** [comment]
Fixed in `v2.2.2`.
